From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Daniel Kiper <daniel.kiper@oracle.com>
Date: Tue, 2 Oct 2018 22:36:43 +0200
Subject: [PATCH] verifiers: Rename verify module to pgp module

Just for clarity. No functional change.

Signed-off-by: Daniel Kiper <daniel.kiper@oracle.com>
Reviewed-by: Ross Philipson <ross.philipson@oracle.com>
(cherry picked from commit b07feb8746c3bb845e3f0d33d37c0bded704d14d)
Signed-off-by: Daniel Axtens <dja@axtens.net>
---
 grub-core/Makefile.core.def            | 4 ++--
 grub-core/commands/{verify.c => pgp.c} | 0
 2 files changed, 2 insertions(+), 2 deletions(-)
 rename grub-core/commands/{verify.c => pgp.c} (100%)

diff --git a/grub-core/Makefile.core.def b/grub-core/Makefile.core.def
index 29c3bf6cd..809f11fea 100644
--- a/grub-core/Makefile.core.def
+++ b/grub-core/Makefile.core.def
@@ -915,8 +915,8 @@ module = {
 };
 
 module = {
-  name = verify;
-  common = commands/verify.c;
+  name = pgp;
+  common = commands/pgp.c;
   cflags = '$(CFLAGS_POSIX)';
   cppflags = '-I$(srcdir)/lib/posix_wrap';
 };
diff --git a/grub-core/commands/verify.c b/grub-core/commands/pgp.c
similarity index 100%
rename from grub-core/commands/verify.c
rename to grub-core/commands/pgp.c
